articles/howtos/create-new-decorator.md

```diff
@@ -0,0 +1,222 @@
+---
+uid: CreateNewDecorator
+---
+# How can I create a new Decorator?
+
+Click [here](xref:Decorator) to find out more about Decorators.
+
+This how-to will serve as an example of how to create a new Decorator that will inject code at runtime for a Template that has been setup for it.
```

Comment:
- Checking on Google, "setup" is a noun, while "set up" is an action, if we kept the wording, then the latter should be used.
- "How to" being mentioned twice in each sentence here feels a bit off to me.
- That article says that how-to's should avoid explanations, so perhaps we don't bother mentioning that code is injected.
- This is more of a global thing, but we need to decide on the style for "Intent" concept words (like Decorator or Template), you had them capitalised, do we want this? Possible alternatives are:
  - Unstyled, just an all lower case word.
  - Italicized, but still lower case.
  - On the first mention of a concept, we hyper link it.

```suggestion
This how-to will guide you through creating new decorator for a template already configured to be able to use it.
```

Reply:
I'm inclined to agree with you about the inject word. We can discuss further tonight.

Reply:
As per my last comment on line 6, adding a link to a the Decorator details.

```suggestion
This how-to will guide you through creating new [](xref:Decorator) for a template already configured to be able to use it.
```

In regards to use of the inject word, having re-read the how-to section of docs [article](https://www.divio.com/blog/documentation/) I linked:
> # FOCUS ON RESULTS
> How-to guides must focus on achieving a practical goal. Anything else is a distraction. As in tutorials, detailed explanations are out of place here.
> # DON’T EXPLAIN CONCEPTS
> A how-to guide should not explain things. It’s not the place for discussions of that kind; they will simply get in the way of the action. If explanations are important, link to them.

I'm thinking again to rather omit the "inject" word, especially as we will have a link now which talks extensively about what a decorator is and what it does. What are your thoughts?